priv/scripts/elixir.sh

```diff
@@ -80,18 +80,19 @@ function upload_build {
 function update_builds_txt {
   otp_versions=(${@:3})
   date=$(date -u '+%Y-%m-%dT%H:%M:%SZ')
+  build_sha256=$(sha256sum mix.exs | cut -d ' ' -f 1)
 
   aws s3 cp s3://s3.hex.pm/builds/elixir/builds.txt builds.txt || true
   touch builds.txt
 
   sed -i "/^${1} /d" builds.txt
   sed -i "/^${1}-otp-\d\+ /d" builds.txt
 
-  echo -e "${1} ${2} ${date}\n$(cat builds.txt)" > builds.txt
+  echo -e "${1} ${2} ${build_sha256} ${date}\n$(cat builds.txt)" > builds.txt
 
   for otp_version in "${otp_versions[@]}"; do
     otp_string=$(otp_string ${otp_version})
-    echo -e "${1}${otp_string} ${2} ${date}\n$(cat builds.txt)" > builds.txt
+    echo -e "${1}${otp_string} ${2} ${build_sha256} ${date}\n$(cat builds.txt)" > builds.txt
```

Comment:
checksum will be different on different OTP versions and we need to take that into account, not sure how to reconcile this.

Reply:
We need to update the txt file for each individual build. So this should remove the `for` loop and be called after this line https://github.com/hexpm/bob/pull/34/files#diff-e1f6d91098b23e308b5c32fc7c7e4735L29.

Reply:
I tried to update the files accordingly. Since the project has many dependencies (S3, Docker etc.), I still can't run it properly without a lot of setup work. Therefore the change is again not tested.